it/it-tests/src/test/java/it/qualityGate/QualityGateTest.java

```diff
@@ -250,6 +251,28 @@ public void ad_hoc_build_break_strategy() throws IOException {
     }
   }
 
+  @Test
+  public void does_not_fail_when_condition_is_on_removed_metric() throws IOException {
```

Comment:
you can reproduce with a medium test, why do you add an IT? We have to be careful to not add to many ITs